Makefile

```diff
@@ -18,7 +18,7 @@ build:
 
 build-appliance:
 	mkdir -p build
-	cd ./cmd && CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o ../build/openshift-appliance
```

Comment:
We should keep the binary naming. I.e. similar to 'openshift-install', we have 'openshift-appliance'.